main/res/xml/preferences_map.xml

```diff
@@ -70,6 +70,29 @@
             android:summary="%s"
             android:title="@string/settings_title_map_rotation"
             app:iconSpaceReserved="false" />
+        <Preference
```

Comment:
Maybe shading could get its own preference category, putting some of the text in "init_mapshading_folder_text" into the category's description?

Reply:
Not sure about this one. Map themes, which is kinda similar, doesn't have its own section either.

I'm also not sure whether pref_mapShadingLinearity and pref_mapShadingScale really need to be user-configurable settings. I've tried to find reasonable defaults for those.
Maybe we could get some feedback during beta whether the defaults are good enough and if they are remove those settings (from GUI at least).

Reply:
Yes, could be a way to go.

Reply:
Let's merge then - your Review is still blocking it somehow

Reply:
> Let's merge then - your Review is still blocking it somehow

Open reviews or reviews requesting changes do not block the build process and do not lead to failing builds.
It's an error in our CI environment, sometimes the server fails with "Cannot create ADB bridge", so it cannot run the tests - which leads to a failed / red build.

Let's wait for @mucek4 to fix this, and retest the PR then.